testing/test_collection.py

```diff
@@ -510,13 +510,8 @@ def test_method(self):
                     pass
         """
         )
-        normid = p.basename + "::TestClass::()::test_method"
-        for id in [
-            p.basename,
-            p.basename + "::TestClass",
-            p.basename + "::TestClass::()",
-            normid,
-        ]:
+        normid = p.basename + "::TestClass::test_method"
+        for id in [p.basename, p.basename + "::TestClass", normid]:
```

Comment:
btw: I would really prefer if black would not make a) the diff unnecessarily bigger here by merging this into a single line, and b) keep it for readability.